mathics/core/symbols.py

```diff
@@ -702,6 +708,29 @@ def atom_to_boxes(self, f, evaluation):
 
 
 class Symbol(Atom):
+    """Note: Symbol is right now used in a couple of ways which in the
+    future may be separated.
+
+    A Symbol a kind of Atom that acts as a symbolic variable or symbolic constant.
```

Comment:
`A Symbol a kind ...` -> `A Symbol is a kind ...`

Reply:
Thanks! Fixed. Other working and grammar typos fixed as well.